runtime/redirector/module.xml

```diff
@@ -64,6 +63,7 @@
 
 	<artifact type="shared" name="jvm_b150" scope="redirector" loadgroup="top" buildlocal="true" appendrelease="false">
 		<include-if condition="spec.flags.module_j9vm"/>
+		<exclude-if condition="spec.java10 or not spec.java9"/>
```

Comment:
I think b150 is obsolete, should it just be deleted?

Reply:
Yes, after the references to the qualified redirectors are gone, we can remove all but the unqualified one.